packages/ag-charts-website/src/stores/frameworkStore.ts

```diff
@@ -33,10 +33,6 @@ export const setInternalFramework = (internalFramework: InternalFramework) => {
         $frameworkContext.setKey('useTypescript', 'false');
     } else if (internalFramework === 'reactFunctionalTs') {
         $frameworkContext.setKey('useTypescript', 'true');
-    } else if (internalFramework === 'vue') {
-        $frameworkContext.setKey('useVue3', 'false');
-    } else if (internalFramework === 'vue3') {
-        $frameworkContext.setKey('useVue3', 'true');
```

Comment:
Should this have been removed? (`useVue3: 'false'` above seems at odds with this too?)

Reply:
Yes - updated to remove the `useVue3` you pointed out